modules/nextflow/src/main/groovy/nextflow/processor/PublishDir.groovy

```diff
@@ -214,6 +223,9 @@ class PublishDir {
         if( params.storageClass )
             result.storageClass = params.storageClass as String
 
+        if( params.retryPolicy )
+            result.retryConfig = new RetryConfig(params.retryPolicy)
+
```

Comment:
Likely as a user, I want to configure this once for all. I'd put something in the config as `nextflow.publish.retryPolicy`